portfolio/src/main/webapp/index-script.js

```diff
@@ -12,15 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const COMMENTS_ID = 'comments-container';
+const MAX_ID = 'max-input';
+const DATA_URL = '/data?max-input=';
+const DELETE_URL = '/delete-data';
+
 /*
  * Fetches message from /data and displays it on the DOM.
  */
 function getComments() {
-  let num = document.getElementById('max-input').value;
-  fetch('/data?max-input=' + num).then(response => response.json()).then((mssg) => {
-    const mssgElem = document.getElementById('comments-container');
+  let num = document.getElementById(MAX_ID).value;
```

Comment:
nit: can be `const`. Not related to anything that's actually changing in this PR, but I tried to find something to comment on...